packages/synapse-react-client/src/components/SetManagedAccessRequirementFields/SetManagedAccessRequirementFields.test.tsx

```diff
@@ -423,19 +455,21 @@ describe('SetManagedAccessRequirementFields', () => {
         MOCK_ACCESS_TOKEN,
         updatedAr,
       )
-      expect(onSaveComplete).toHaveBeenCalledTimes(1)
-      expect(onSaveComplete).toHaveBeenLastCalledWith(updatedAr)
+      expect(onSave).toHaveBeenCalledTimes(1)
+      expect(onError).not.toHaveBeenCalled()
     })
   })
 
   test('displays an error when expiration period is < 0', async () => {
     const negativeExpirationPeriodDays = '-25'
-    const { user, expirationPeriodInput, ref } = setUp()
+    const { user, expirationPeriodInput, ref } = await setUp()
 
     await user.clear(expirationPeriodInput)
     await user.type(expirationPeriodInput, negativeExpirationPeriodDays)
+    await user.tab() // trigger blur
```

Comment:
Expiration period error is now displayed on blur, rather than waiting for user to click save in the parent